codegenerator/cli/templates/static/codegen/src/EventRouter.res

```diff
@@ -85,7 +85,7 @@ let get = (router: t<'a>, ~tag, ~contractAddress, ~contractAddressMapping) => {
   }
 }
 
-let getEvmEventTag = (~sighash, ~topicCount) => {
+let getEvmEventId = (~sighash, ~topicCount) => {
```

Comment:
Looks like this is now dead code?

Reply:
Oh nvmnd it will be used for routing

Reply:
Yeah, I also removed it with such thought, but then noticed it used for routing :)